src/vs/workbench/parts/terminal/electron-browser/terminalActions.ts

```diff
@@ -29,9 +29,32 @@ import { TERMINAL_COMMAND_ID } from 'vs/workbench/parts/terminal/common/terminal
 import { Command } from 'vs/editor/browser/editorExtensions';
 import { timeout } from 'vs/base/common/async';
 import { FindReplaceState } from 'vs/editor/contrib/find/findState';
+import { execSync } from 'child_process';
+import { isWindows } from 'vs/base/common/platform';
 
 export const TERMINAL_PICKER_PREFIX = 'term ';
 
+function getCwd(configHelper: ITerminalConfigHelper, activeInstance: ITerminalInstance): string {
+	switch (configHelper.config.splitCwdSource) {
+		case 'ShellDefault': {
+			// allow default behavior
+			return '';
```

Comment:
Should this be `null` or `undefined` instead? Note that I just added this file to strict null checks so make sure you sync and run `yarn strict-null-check` to make sure we keep passing.

Reply:
The default value for cwd in the shell config is empty. If this is changed to null or undefined then we have to add checks for it.